geosyspy/geosys.py

```diff
@@ -257,24 +126,24 @@ def get_time_series(self, polygon, start_date, end_date, collection, indicators)
         Raises:
             ValueError: The collection doesn't exist
         """
-
-        if collection in self.list_collection_weather:
+        # if collection in self.list_collection_weather:

```

Comment:
From my PoV commented code is useless because in 99% cases it'll become dead code.

Reply:
I totally agree, got rid of unnecessary comment!